wazo_confd/plugins/user/schema.py

```diff
@@ -50,6 +50,7 @@ class UserSchema(BaseSchema):
     lastname = fields.String(validate=Length(max=128), allow_none=True)
     email = fields.Email(validate=Length(max=254), allow_none=True)
     timezone = fields.String(validate=Length(max=128), allow_none=True)
+    country = fields.String(dump_only=True)
```

Comment:
Should we modify the [unpaginated_user_strategy](https://github.com/wazo-platform/xivo-dao/blob/master/xivo_dao/resources/user/strategy.py#L7) to joinedload the country?

Reply:
Yes, we added the `tenant` table to the strategy: https://github.com/wazo-platform/xivo-dao/pull/278/commits/a96cb6fa0aef9f6c3d68ef83ad4ea6e45745fb92

Thanks for the comment :)